File: src/test/data/java-packaged-asst/submissions/whitespace/sqrtProg.java
public class sqrtProg {

    private double x;

    public sqrtProg(double xx) {
        x = xx;
    }

    public void printSqrt()
    {
        double root = Math.sqrt(x);
        System.out.printf("The square\troot of %.2f is %.4f.%n",
             x, root);
    }

    static public void main(String[] args) {
        if (args.length != 1) {
            System.err.println ("Usage: java sqrtProg x");
            System.err.println ("  where x is a non-negative number.");
            System.exit(1);
        }
        double x = Double.parseDouble(args[0]);
        if (x < 0.0) {
            System.err.println ("Number must be non-negative");
            System.exit(2);
        }
        new sqrtProg(x).printSqrt();
    }
}

File: demos/standAlone/sqrtProg.cpp
#include <iostream>
#include <cmath>
#include <cstdio>

using namespace std;

void printSqrt(double x)
{
    double root = sqrt(x) + ((x < 1.0) ? .0002 : 0);  // bug: affects only small roots
    printf("The square root of %.2f is %.4f.\n",
           x, root);
}

int main(int argc, char **argv)
{
    if (argc != 2)
    {
        cerr << "Usage: " << argv[0] << " x" << endl;
        cerr << "  where x is a non-negative number." << endl;
        return 1;
    }
    double x = atof(argv[1]);
    if (x < 0.0)
    {
        cerr << "Number must be non-negative" << endl;
        return 2;
    }
    printSqrt(x);
}
